### DISCUSSION TITLE
Not able to build podman from source

### DISCUSSION THREAD

**Discussion by itsCheithanya** [containers/podman#24111] [Q&A] [answered]
This was the issue when installing podman from source `Error: could not find "netavark" `
Even though i installed `containernetworking-plugins` using` sudo apt-get install containernetworking-plugins `
```
cheithanya@cheithanya-lenovo-v14-ikb:~/Desktop/podman$ sudo apt-get install containernetworking-plugins
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
containernetworking-plugins is already the newest version (0.9.1+ds1-1).
The following packages were automatically installed and are no longer required:
  docker-scan-plugin libllvm13
Use 'sudo apt autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 152 not upgraded.
```
 
```
cheithanya@cheithanya-lenovo-v14-ikb:~/Desktop/podman$ sudo podman version
Error: could not find "netavark" in one of [/usr/local/libexec/podman /usr/local/lib/podman /usr/libexec/podman /usr/lib/podman].  To resolve this error, set the helper_binaries_dir key in the `[engine]` section of containers.conf to the directory containing your helper binaries.
```

**Comment by Luap99** [comment, marked as answer]
podman 5 dropped CNI support, you need to use netavark and aardvark-dns for networking

**Reply by afbjorklund** [reply]
Documentation still says "The netavark package may not be available on older Debian / Ubuntu versions. Install the containernetworking-plugins package instead.", where it probably should instruct to install netavark from source?

https://podman.io/docs/installation#building-from-source

It also only shows how to install golang, not how to install rust

**Reply by itsCheithanya** [reply]
Yes the documentation is not updated then to guide us on how to install netavark and aardvark-dns using rust,maybe @Luap99 the podman team can update it

**Reply by afbjorklund** [reply]
There is a similar issue with pasta (as compared to slirp4netns), so I think that the team needs help with documentation...

* https://github.com/containers/podman.io/issues/285

**Reply by pierluigizagaria** [reply]
What is the solution for who compiled from source?

**Reply by afbjorklund** [reply]
See https://github.com/containers/netavark and https://passt.top
